test/commands/validate.test.ts

```diff
@@ -34,6 +34,16 @@ describe('validate', () => {
         expect(ctx.stderr).to.equals('ValidationError: There is no file or context with name "./test/not-found.yml".\n');
         done();
       });
+
+    test
+      .stderr()
+      .stdout()
+      .command(['validate', 'https://bit.ly/asyncapi'])
```

Comment:
I see that @fmvilas already reviews, so from my point of view I only suggest to log a followup issue, something good for the first contribution, to make sure tests do not rely on external resources. Basically, external resources are risky to use in tests and can cause tests to be flaky, like randomly failing just because the external resource is not available. So follow up should describe that tests should actually start independent server hosting asyncapi file, like we do in parser -> https://github.com/asyncapi/parser-js/blob/master/package.json#L20